comments/tests/test_models.py

```diff
@@ -0,0 +1,39 @@
+from datetime import timedelta
+
+from django.test import TestCase
+from django.contrib.auth.models import User
```

Comment:
missed this, but noticed later; don't use the User model directly! :)

gotta use `from django.contrib.auth import get_user_model` and then `User = get_user_model()`